app/client/packages/design-system/ads/src/Templates/EntityExplorer/EntityListTree/EntityListTree.tsx

```
@@ -0,0 +1,82 @@
+          <EntityItemWrapper
+            alignItems="center"
+            aria-expanded={item.isExpanded}
+            aria-level={currentDepth}
+            aria-selected={item.isSelected}
+            data-depth={currentDepth}
+            data-disabled={item.isDisabled || false}
+            data-selected={item.isSelected}
+            flexDirection="row"
+            role="treeitem"
+          >
+            {item.children && item.children.length ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)